### PR TITLE
Monalisha_SoliHull Assignment

### PR DESCRIPTION
**Pull request by monalisha-90**
This is the assignment for SoliHull using Robot framework
Please review and approve.

### CODE REVIEW

File: Monalisha/Resources/Pages/SoliHull_page.robot [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Garima2412) [inline Monalisha/Resources/Pages/SoliHull_page.robot]: Kindly remove Unused spaces

File: Monalisha/Resources/Solihull_SD.robot [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+I click on top right corner dropdown
+    click on right most dropdown
+
+I print page title of selected languages starting with letter "${StartLetter}"
+    count language starts with "${StartLetter}"
+    select language and print page title
+
+
+
+
```
Comment (Garima2412) [inline Monalisha/Resources/Solihull_SD.robot]: Kindly remove extra spaces

File: Monalisha/Resources/Pages/SoliHull_page.robot [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+#        Wait Until Element Is Not Visible      ${DropDownexpandXpath}            20sec
+        Wait Until Page Contains Element        ${CloseDropdownXpath}         10sec
+        ${currentpagetitle}     get title
+        log to console                      ${\n}
+        log to console          Selected language is: ${language} and title is: ${currentpagetitle}
+#        Wait Until Element Is Not Visible      ${DropDownexpandXpath}            5sec
+        click on right most dropdown
+#        Wait Until Element Is Visible         ${DropDownexpandXpath}            5sec
+        Wait Until Page Contains Element        ${OpenDropdownXpath}          10sec
+        Exit For Loop if            ${i}==${SelectedLangCount}
```
Comment (Garima2412) [inline Monalisha/Resources/Pages/SoliHull_page.robot:56]: We can omit the if condition here,
Exit for loop condition is not required as the loop is working perfectly with in the apt range calculated above.

**Review by narendrashakya — DISMISSED** [review]
Could you Please remove other member folder from the branch. Only your name folder is required.
Rest is good to merge. Thanks.

image

**Review by mssquaretech — APPROVED** [review]
Ok to Approve

File: Monalisha/Resources/Pages/SoliHull_page.robot (hunk 1 above)
Comment (monalisha-90) [reply Monalisha/Resources/Pages/SoliHull_page.robot]: @Garima2412 : update the expected changes. Please review and approve.
Thanks.

File: Monalisha/Resources/Solihull_SD.robot (hunk 2 above)
Comment (monalisha-90) [reply Monalisha/Resources/Solihull_SD.robot]: @Garima2412 : updated expected changes. Please review and approve.
thanks

**Review by narendrashakya — APPROVED** [review]
Good to merge.

**Pull request merged**